apps/mull-ui/src/app/pages/create-event/create-event.scss

```diff
@@ -0,0 +1,90 @@
+@import '../../../variables.scss';
+
+.create-event {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+
+  .custom-text-input-container {
+    margin-bottom: 30px;
+  }
+}
+
+.event-image-upload {
+  display: none;
+}
+
+.event-input-border {
+  border-radius: 20px;
+  border: 1.5px #27b09a solid;
+  &.error {
+    border-color: #ff0000;
+  }
+}
+
+.custom-file-upload {
+  display: block;
+  padding: 1.8em 5em;
+  cursor: pointer;
+
+  margin-bottom: 20px;
+
+  &:hover {
+    background: #0ba024;
+  }
+}
+
+.create-event-text {
+  color: $primary-color;
+}
+
+.create-event-label {
+  color: black;
+  align-items: flex-start;
+  font-weight: 600;
+}
+
+.create-event-time {
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+
+  .custom-time-picker-container {
+    margin: 0 0 1.25rem;
+  }
+
+  @media (min-width: $breakpoint-mobile) {
+    flex-direction: row;
+    justify-content: space-evenly;
+    margin: 0 0 1.25rem;
+  }
+}
+
+.create-event-input {
+  display: block;
+  box-sizing: border-box;
+  outline: none;
+  padding-left: 1em;
+  width: 100%;
+  height: 2.8em;
+}
+
+.create-event-button {
+  color: white;
+  font-weight: bold;
+  padding: 15px 55px;
+  background: $primary-color;
+  border: 1px solid $primary-color;
+  border-radius: 9px;
+  outline: none;
+  margin-top: 20px;
+}
+
+.create-event-fields {
+  width: 100%;
+
+  @media (min-width: $breakpoint-mobile) {
+    width: 70%;
+  }
```

Comment:
Move this into the single  `@media (min-width: $breakpoint-mobile)` I mentioned before.

example of what it should look like
```
@media (min-width: $breakpoint-mobile) {
  .create-event-fields {
    width: 70%;
  }
}
```